File: src/main/java/net/kenro/ji/jin/purescript/psi/impl/PSBinderImpl.java
package net.kenro.ji.jin.purescript.psi.impl;

import com.intellij.lang.ASTNode;
import net.kenro.ji.jin.purescript.psi.ContainsIdentifier;


public class PSBinderImpl extends PSPsiElement implements ContainsIdentifier {

    public PSBinderImpl(final ASTNode node) {
        super(node);
    }

}

File: src/main/java/net/kenro/ji/jin/purescript/psi/impl/PSBinderImpl.kt
package net.kenro.ji.jin.purescript.psi.impl;

import com.intellij.lang.ASTNode;
import net.kenro.ji.jin.purescript.psi.DeclaresIdentifiers;


public class PSBinderImpl extends PSPsiElement implements DeclaresIdentifiers {

    public PSBinderImpl(final ASTNode node) {
        super(node);
    }

}
